src/Drivers/CraftTinkerwellDriver.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Tinkerwell\ContextMenu\Label;
+use Tinkerwell\ContextMenu\OpenURL;
+use Tinkerwell\ContextMenu\Separator;
+
+class CraftTinkerwellDriver extends TinkerwellDriver
+{
+    private $version;
+
+    public function canBootstrap($projectPath)
+    {
+        return file_exists($projectPath . '/web/index.php') &&
+            file_exists($projectPath . '/craft');
+    }
+
+    public function bootstrap($projectPath)
+    {
+        // Set path constants
+        define('CRAFT_BASE_PATH', $projectPath);
+        define('CRAFT_VENDOR_PATH', CRAFT_BASE_PATH.'/vendor');
+
+        // Load Composer's autoloader
+        require_once CRAFT_VENDOR_PATH.'/autoload.php';
+
+        // Load dotenv?
+        if (file_exists(CRAFT_BASE_PATH.'/.env')) {
+            (new Dotenv\Dotenv(CRAFT_BASE_PATH))->load();
```

Comment:
I just tried this locally - this isn't working (at least with the latest version of craft).
Shouldn't it be this:
```php
Dotenv\Dotenv::create(CRAFT_BASE_PATH)->load();
```

Reply:
Odd… it's working for me, but you're right—the dotenv loader was from an outdated version of Craft's [console command](https://github.com/craftcms/craft/blob/master/craft). I've updated it and tested on a fresh install. If it still doesn't work for you can you tell me what error you're getting?

Reply:
Ah, I just did the logical thing and switched back to my original version on my clean install of Craft 3.4.4.1. And now I see what you were probably seeing:

```
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Dotenv\Dotenv::__construct() must be an instance of Dotenv\Loader, string given…
```